atomic_reactor/inner.py

```diff
@@ -337,6 +341,7 @@ def build_docker_image(self):
             start_time = datetime.datetime.now()
             self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
 
+            signal.signal(signal.SIGTERM, self.throw_cancelled_build_exception)
```

Comment:
Do you think this should be prior to prebuild plugins? I think for now there's nothing in there that needs to be cleaned up, but not sure if it'll always be the case.